src/Surfnet/ServiceProviderDashboard/Application/ViewObject/EntityList.php

```diff
@@ -36,15 +36,41 @@ public function __construct(array $entities)
     }
 
     /**
+     * @param String $sortBy
      * @return Entity[]
      */
-    public function getEntities()
+    public function getEntities(String $sortBy = '')
     {
+        if (strlen($sortBy) > 0) {
+            $entities = $this->entities;
+            switch ($sortBy) {
+                case 'environment':
```

Comment:
Please add a class constant for this value. Also, I'm not stoked on adding the sorting logic to a getter. A getter should be dumb as possible. Maybe decouple this from the getter and add a (chainable) sort method? That way, the other getters could also benefit from the sorting logic. And the getter has a single responsibility.

Reply:
i like the idea of a chainable sort method, brilliant 💯